### PR TITLE
Closes #1776: Fixed other terms tab.

### LINKED ISSUE #1776: Fix mess on "Other licenses" tab
* The tab should be labelled "Other terms" - these are not licenses. 
* Something is wrong with the display of other terms: 

image

* The last column should be labeled "Active" (NOT "Active by Default") - same thing with the column name on "Licenses" tab

### PR DESCRIPTION
**Pull request by MrK191**

### CODE REVIEW

File: dataverse-webapp/src/main/webapp/resources/vecler/theme-base.scss [hunk 1]
```diff
@@ -1725,7 +1725,7 @@ div.filesTable {
 	}
 }
 
-@media screen and (min-width: $screen-md-min) {
+@media screen and (max-width: $screen-md-min) {
```
Comment (madryk) [inline dataverse-webapp/src/main/webapp/resources/vecler/theme-base.scss]: You're change will do current things:
 - On wide device table will render in full width
 - On small device table will render in 50% of full width

I don't think that is the thing that we want to achieve.
If we think that table should always render in full width then this whole rule is unnecessary.
To manipulate whether table is scrollable we should add css class to datatable:
`no-min-width` - table will not cause scrolling to appear at all
`medium-min-width` - table will cause scrolling, but it let to be a little smaller (breakpoint for scroll has smaller `width`)
If we want some more options then introducing new class is required (analogous to medium-min-width)

**Review by madryk — CHANGES_REQUESTED** [review]

File: dataverse-webapp/src/main/webapp/resources/vecler/theme-base.scss (hunk 1 above)
Comment (MrK191) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme-base.scss]: Well, I didn't know what was the idea behind this modification so I assumed that maybe the parameter was reversed. The task is also not that specific so I assumed that the whole width will be fine. So now I'm not sure whether to just remove the rule or to also take care of scrolling.

Comment (madryk) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme-base.scss]: Well, experiment with it. Table doesn't require much space so `no-min-width` maybe will be a good choice. As for if it should always display in full width - I don't have strong opinion - maybe if it causes more problems then it resolves then setting it to always display in full width is not a bad idea.

Btw. in task there is also point: `The last column should be labeled "Active" (NOT "Active by Default") - same thing with the column name on "Licenses" tab` which is not addressed in this pr

Comment (madryk) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme-base.scss]: Ok. I see that we went for full width + `no-min-width`

File: dataverse-webapp/src/main/webapp/dashboard-licenses.xhtml [hunk 2]
```diff
@@ -94,7 +94,7 @@
                     </p:tab>
 
                     <p:tab title="#{bundle['dashboard.otherTermsOfUse.header']}">
-                        <p:dataTable var="otherTos" class="table-othertos" value="#{OtherTermsOfUseTab.otherTermsOfUseDto}">
+                        <p:dataTable var="otherTos" class="table-othertos no-min-width" value="#{OtherTermsOfUseTab.otherTermsOfUseDto}">
```
Comment (madryk) [inline dataverse-webapp/src/main/webapp/dashboard-licenses.xhtml:97]: From the task description:
`The last column should be labeled "Active" (NOT "Active by Default") - same thing with the column name on "Licenses" tab` - this is not addressed in this pr

**Review by madryk — CHANGES_REQUESTED** [review]

File: dataverse-webapp/src/main/webapp/dashboard-licenses.xhtml (hunk 2 above)
Comment (MrK191) [reply dataverse-webapp/src/main/webapp/dashboard-licenses.xhtml:97]: Oh I haven't noticed there is something under the picture

**Review by madryk — APPROVED** [review]

**Pull request merged**